charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -31,11 +31,8 @@ spec:
         {{- end }}
     spec:
       serviceAccountName: {{ include "trino.serviceAccountName" . }}
-      {{- with .Values.securityContext }}
       securityContext:
-        runAsUser: {{ .runAsUser }}
-        runAsGroup: {{ .runAsGroup }}
-      {{- end }}
+        {{- toYaml .Values.securityContext | nindent 8 }}
```

Comment:
According to #181, it might be a good idea to put the whole `securityContext` in a condition block, in case it's empty.

Reply:
If I add `securityContext: ~` to a values file, the template renders as `securityContext: null` which then gets applied as `securityContext: {}`. It seems to handle that situation. If you think a condition block reads better, I'll add it in.

Reply:
So we don't need a condition block, thanks! Can you document that in the comment for this section?

Reply:
I tested this and it doesn't work with either `~`, empty string or an empty object. Please put this in a conditional block.

Reply:
Hmm, I wonder why I wasn't able to reproduce the the issue. Yeah, I'll put in the conditional sometime today.

Reply:
@nineinchnick I've put the `with` condition back in and also added the ability to configure the security context for the jmx exporter container.

Reply:
Interesting, it's now possible to clear the `securityContext` with `~`, but not with `{}`. Can you either document it, or change the `with` block into an `if`?

Reply:
Alright, I've updated the documentation.